recipes/asdf-standard/meta.yaml

```
@@ -0,0 +1,47 @@
+{% set name = "asdf-standard" %}
+{% set version = "1.0.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/asdf_standard-{{ version }}.tar.gz
+  sha256: 1caf0671e126f4aec3113ade1b5063858369a769255cfbbbbcd2d67dacc5b592
+
+build:
+  skip: true  # [py<37]
```

Since this is pure Python, it would be better to make this noarch and unconditionally depend on importlib-resources below:
```suggestion
  noarch: python
```